src/main/java/org/folio/consortia/domain/dto/SourceValues.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.consortia.domain.dto;
+
+public enum SourceValues {
+  CONSORTIUM_FOLIO_INSTANCE("CONSORTIUM-FOLIO"),
+  CONSORTIUM_MARC_INSTANCE("CONSORTIUM-MARC"),
+  CONSORTIUM("consortium"),
+  USER("user");
+
+  private final String value;
+
+  SourceValues(String value) {
```

Comment:
The constructor and the getter on the Enum can replaced with `@Getter` and `@AllArgsConstructor` annotations

Reply:
Thanks to suggestion, improved it